### ISSUE TITLE
Different Splah Page Photo Based on Location - Low

### ISSUE THREAD

**Issue by Happs80** [humansis/front#168]
For example, Cambodia login is presented with Cambodia specific photo.

**Comment by blazekv** [comment]
Do you have any ideaon which information do you want to change that splash screen? GPS detection looks like overkill for me. It could be done via browser language.

For example for specific languages special splash screen for everything else default splash screen. If it is OK. I will need languages codes and splash screens for them.

If you would like to test your browser language you could do that in developer tools (Console) by pasting
 `console.log(navigator.language || navigator.userLanguage);`

**Comment by Happs80** [comment]
Is it possible based on ip range?

**Comment by blazekv** [comment]
I dont think that could be easily done. Because frontend application runs on client device (so it does not have idea which public IP user have). It could be done via services (like this one `https://api.ipify.org/?format=json`) or backend calls, but still it seems litle bit overkill.

If you need it based on IP you have probably two options:

1) Call ipify.org (system will be dependent on other service) of course there will be fallback to default if ipify.org fails.
2) Create backend service which will return country based on IP. It will be much better to define IP ranges on backend and just return country based on detected IP.

But defining IP ranges and finding correct range still seems as too much work for nice to have feature.

**Comment by Happs80** [comment]
Ok I think you're right, this is too much work.

What about just a rotating list of randomised images instead?

**Comment by blazekv** [comment]
I have a some idea, but dont know if it is good enough. I could do that. It is easy. Could you please send me all images you want to rotate? Current image has resolution 1618x1080
